### PR TITLE
Small text changes

### PR DESCRIPTION
**Pull request by rob-luke**

### CODE REVIEW

File: src/spec/create_extension_spec.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ def main():
                 name="frequency",
                 doc=(
                     "The modulation frequency in Hz used for frequency domain NIRS."
```
Comment (rly) [inline src/spec/create_extension_spec.py:213]:
```suggestion
                    "The modulation frequency in Hz used for frequency domain NIRS"
```

Comment (dsleiter) [reply src/spec/create_extension_spec.py:213]: Good catch on the erroneous sentence @rob-luke and @rly.

Looking at the docs for neighboring attributes, I think the original text was missing an "Only used".

```suggestion
                    "The modulation frequency in Hz used for frequency domain NIRS."
                    " Only used if nirs_mode is a type of frequency domain spectroscopy."
```

**Review by dsleiter — CHANGES_REQUESTED** [review]
Thanks for the doc updates @rob-luke! I had one small requested change, but the rest looks good.

File: src/spec/create_extension_spec.py (hunk 1 above)
Comment (rob-luke) [reply src/spec/create_extension_spec.py:213]: Thank you both, this makes more sense now.

**Comment by rob-luke** [comment]
Thanks for the feedback @rly and @dsleiter

**Review by dsleiter — APPROVED** [review]
Thanks, @rob-luke, as we discussed I pushed the small fix and will now merge!

**Pull request merged**